src/main/resources/native/android/dalvik/MainActivity.java

```diff
@@ -228,5 +255,239 @@ protected void onStop() {
         super.onStop();
     }
 
+    public final static int PRESS   = 111;
+    public final static int RELEASE = 112;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4825537)